app/views/admin/cms/show.html.erb

```diff
@@ -0,0 +1,23 @@
+<% content_for :cms_heading do %>
```

Comment:
I was playing around with the layout structure to explicitly define where the list and possibly the collection form could render beside each other.